Libraries/Components/View/View.js

```diff
@@ -167,8 +169,8 @@ const View: React.AbstractComponent<
               : importantForAccessibility
           }
           nativeID={id ?? nativeID}
-          {...restWithDefaultProps}
           style={style}
+          {...restWithDefaultProps}
```

Comment:
This must be before `style` because it is now part of the rest props and will replace the flattened style created above. The kind of regression that would ideally have unit tests to catch